pfb-analysis/scripts/setup_database.sh

```diff
@@ -6,11 +6,18 @@ NB_POSTGRESQL_DB=pfb
 NB_POSTGRESQL_USER=gis
 NB_POSTGRESQL_PASSWORD=gis
 
+# Set configuration parameters
+su postgres bash -c psql <<EOF
+ALTER SYSTEM SET work_mem TO '4096MB';
+ALTER SYSTEM SET checkpoint_completion_target TO 0.8;
+ALTER SYSTEM SET max_wal_size TO '5GB';
```

Comment:
This setting also causes adverse effects in dev and probably will also do so in production because we'll be a bit limited on storage space there too, assuming we can reconfigure as described below.

This setting allows the WAL files to grow to 5Gb. Docker, when using the `devicemapper` storage driver, defaults to volumes capped at 10G. So we use up half of our default available storage space holding WAL files instead of the analysis data. I haven't tested if theres a performance hit to leaving the `max_wal_size` low, but that's almost certainly preferable to using up 5G of storage in dev for transaction logs, which we don't care about. Doubly so when the VM volume is limited to 40G total without significant, not automatically provisionable workarounds.

Either way, I'm not quite sure yet of two things:
- Whether we want this setting at all, since it balloons disk usage at the expense of (maybe) a slight performance increase
- Whether this is the root cause of the analysis run failures I'm working on in #109